249 Updated terms related to BIOS and firmware (#250)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

[discrete]
[[bios]]
==== image:images/yes.png[yes] BIOS (noun)

> *Description*: "BIOS" is an abbreviation for basic input and output system. The plural form is "BIOSes". BIOS is the specific name for the motherboard firmware in older PCs. Modern computers have a different kind of firmware, called either EFI or UEFI. If you need to refer to computer firmware, do not use BIOS as a general term. Use _firmware_ or a specific phrase like "UEFI firmware or legacy BIOS" instead.

Reviewer: ```suggestion
*Description*: "BIOS" is an abbreviation for Basic Input/Output System. The plural form is "BIOSes". BIOS is the specific name for the motherboard firmware that provides runtime services for operating systems in older PCs. Modern computers use a different kind of firmware, called either EFI or UEFI. 

Do not use _BIOS_ as a generic term to refer to this type of firmware. Use _firmware_ or a specific phrase like "UEFI firmware" or "legacy BIOS" instead.
```
- Should be "Basic Input/Output System". See https://en.wikipedia.org/wiki/BIOS.
- A few other suggestions to tighten things up a bit.

[discrete]
[[bios]]
==== image:images/yes.png[yes] BIOS (noun)
*Description*: "BIOS" is an abbreviation for Basic Input/Output System. The plural form is "BIOSes". BIOS is the specific name for the motherboard firmware that provides runtime services for operating systems in older PCs. Modern computers use a different kind of firmware, called either EFI or UEFI. 

> Do not use _BIOS_ as a generic term to refer to computer firmware. Use _firmware_ or a specific phrase like "UEFI firmware" or "legacy BIOS" instead.

Reviewer 2: Maybe also expand EFI and UEFI on first occurrence?

> Do not use _BIOS_ as a generic term to refer to computer firmware. Use _firmware_ or a specific phrase like "UEFI firmware" or "legacy BIOS" instead.

Author: Noted on the IBM style rec -- thanks!
Now I'm wondering if introducing EFI and UEFI requires entries or guidance on whether to expand the acronyms if you're using them (in non-SSG content). Any thoughts?

> Do not use _BIOS_ as a generic term to refer to computer firmware. Use _firmware_ or a specific phrase like "UEFI firmware" or "legacy BIOS" instead.

Reviewer 2: I would just treat them as normal (not well known) abbreviations, and expand them on first occurrence.

> Do not use _BIOS_ as a generic term to refer to computer firmware. Use _firmware_ or a specific phrase like "UEFI firmware" or "legacy BIOS" instead.

Reviewer: Like BIOS, EFI and UEFI are hardly ever used in expanded forms that I have encountered. That being said, it might be a good idea to create separate entries for them in a separate PR, but I would argue against expanding them here.

> Do not use _BIOS_ as a generic term to refer to computer firmware. Use _firmware_ or a specific phrase like "UEFI firmware" or "legacy BIOS" instead.

Reviewer 2: Fair enough -- I'm not familiar with the area so I'll defer to Brian's expertise :)

> Do not use _BIOS_ as a generic term to refer to computer firmware. Use _firmware_ or a specific phrase such as "UEFI firmware" or "legacy BIOS" instead.

Reviewer 3: Hi @sbmetz, t his PR looks good to me but just one question on this bit: Shouldn't BIOS and firmware be in quotations here because it's not within the definition of the terms but rather it's about how to write them (or not in this case). Looking at the contributing guidance that was added recently:

![image](https://user-images.githubusercontent.com/30370363/198079041-4baa4ded-46e1-48b0-adb3-fc23ff98ce80.png)

> Do not use _BIOS_ as a generic term to refer to this type of firmware. Use _firmware_ or a specific phrase like "UEFI firmware" or "legacy BIOS" instead.

Reviewer 4: ```suggestion
Do not use _BIOS_ as a generic term to refer to computer firmware. Use _firmware_ or a specific phrase like "UEFI firmware" or "legacy BIOS" instead.
```

Reviewer: I'm good with this Aiden's suggestion. Otherwise, all looks good to me
